docs/integrations/new-editor.md

```diff
@@ -1062,3 +1062,21 @@ interface FindTextInDependencyJarsRequest(
 _Response_:
 
 - result: `Location[]`
+
+### `metals/doctorVisibilityDidChange`
+
+When the client opens the doctor view, it will send in `visible: true` to the
```

Comment:
out of curiosity, is `in` needed in `send in to the server`?

Reply:
ha, funny questions. Both would be correct. They essentially mean exactly the same thing in this scenario. I can remove it if you think it reads better.

Reply:
Lets keep it, I was just curious ;)